[2022-11-28 19:34:02,314] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: hello_world_dag.hello_world_task scheduled__2022-11-28T19:33:00+00:00 [queued]>
[2022-11-28 19:34:02,325] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: hello_world_dag.hello_world_task scheduled__2022-11-28T19:33:00+00:00 [queued]>
[2022-11-28 19:34:02,325] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-11-28 19:34:02,325] {taskinstance.py:1244} INFO - Starting attempt 1 of 4
[2022-11-28 19:34:02,326] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-11-28 19:34:02,338] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): hello_world_task> on 2022-11-28 19:33:00+00:00
[2022-11-28 19:34:02,342] {standard_task_runner.py:52} INFO - Started process 1656 to run task
[2022-11-28 19:34:02,345] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'hello_world_dag', 'hello_world_task', 'scheduled__2022-11-28T19:33:00+00:00', '--job-id', '439', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpq22kznnd', '--error-file', '/tmp/tmpmv5j1w15']
[2022-11-28 19:34:02,346] {standard_task_runner.py:77} INFO - Job 439: Subtask hello_world_task
[2022-11-28 19:34:02,417] {logging_mixin.py:109} INFO - Running <TaskInstance: hello_world_dag.hello_world_task scheduled__2022-11-28T19:33:00+00:00 [running]> on host acb48b0a6e0d
[2022-11-28 19:34:02,488] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Ranga
AIRFLOW_CTX_DAG_ID=hello_world_dag
AIRFLOW_CTX_TASK_ID=hello_world_task
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T19:33:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T19:33:00+00:00
[2022-11-28 19:34:02,489] {python.py:175} INFO - Done. Returned value was: Hello World!
[2022-11-28 19:34:02,514] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=hello_world_dag, task_id=hello_world_task, execution_date=20221128T193300, start_date=20221128T193402, end_date=20221128T193402
[2022-11-28 19:34:02,559] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-11-28 19:34:02,595] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
